kuma/users/jinja2/socialaccount/includes/google_analytics.html

```diff
@@ -0,0 +1,18 @@
+<script>
+    // Mozilla DNT Helper
+    {% include "js/libs/mozilla.dnthelper.min.js" %}
+    {% - if settings.GOOGLE_ANALYTICS_ACCOUNT %}
+    {% - set GA_FILE = 'analytics_debug' if settings.DEBUG else 'analytics' %}
```

Comment:
The space between `{%` and `-` on these two lines as well as line 17 below (all should be `{%-` rather than `{% -`) is what is causing the `make localeextract` errors detailed [here](https://github.com/mdn/kuma/pull/6057#issuecomment-547327318), but as noted above, this entire file is redundant and should be removed.

Reply:
I bet I missed a Cmd+K+S and `prettier` did that. Interesting, thanks Ryan.